profile/README.md

```diff
@@ -135,3 +135,5 @@ See [Vision](https://github.com/openebs/community/blob/develop/VISION.md) for mo
 ## New Contributors and Maintainers
 We are actively looking for new contributors and maintainers. Want to get involved, but not sure how? Please email the  maintainers at openebs.team@gmail.com
 
+## OpenEBS Community Code of Conduct
+OpenEBS follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
```

Comment:
Shouldn't it point to the community repo like we do here:
https://raw.githubusercontent.com/openebs/openebs/refs/heads/main/CODE_OF_CONDUCT.md

Reply:
Not here. It's expected to explicitly point to the CNCF code of conduct. I've checked on other projects like Kusionstack.

Reply:
Not sure why would it be different here, what do you mean by "it's expected"? Is this coming from a guideline?

Reply:
`Ensure that that the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's README on GitHub.`

This is from the onboarding ticket.

Reply:
I got no reply from CNCF, but seems most projects do this, so let's just follow suit.